### PR TITLE
Added screencast tutorials to docs with link to youtube

### LINKED ISSUE #795: Determine best hosting for LibreTime Tutorials
**Is your feature request related to a problem? Please describe.**
I have developed a number of screencast tutorials that explain various features and I'd like them to be officially hosted by the project.

**Describe the solution you'd like**
We could setup a 3rd party host such as youtube or archive.org and then embed into mkdocs by modifying the javascript as described [here](https://github.com/mkdocs/mkdocs/issues/243) 

**Describe alternatives you've considered**
We could also setup a separate repo called tutorials on git and transcode them to webm and mp4 and include them video the <video> tag. Adding them to the main docs repo would increase the size of the main git repo too much.

**Additional context**
Currently the videos are unlisted on youtube on my personal account and available via the [wiki](https://github.com/LibreTime/libretime/wiki/Tutorials)

I would also like to credit the [Greater Columbus Arts Council](https://gcac.org) in the videos since they provided via a grant to my organization to pay me to develop documentation. We don't have a formal mechanism for mentioning sponsors but I'd like us to discuss that.

### PR DESCRIPTION
**Pull request by Robbt**
So I added the all of the tutorials with links to the newly created LibreTime youtube channel. 
I used the no-cookies link so that they will play without opening up in people's youtube account. 
If people think that we should switch to linking directly to the youtube page that would be fun. 
I wasn't able to embed the videos directly in the page so I just took the screenshot and linked to the video.

This fixes #795 as we are on youtube officially for now. We can add alternative hosting as the need or motivation arises.

### CODE REVIEW

**Comment by Robbt** [comment]
I also added a number of how-tos that were primarily derived from the tutorial scripts but broken down to step-by-step instructions. 

The next step is to link to add screen-shots to the how-to pages and link to them from other relevant manual sections.

File: docs/manual/howtos/build-schedule.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Click **Repeats **if this show will air more than once.
+
+If it airs at the same time of the day on more than one day of the week
+then check the boxes next to the Days it should repeat on.
+
+If this is a limited engagement show uncheck **No End?** and select the
+date at which this show should no longer be on the schedule.
+
+**Autoloading Playlist **is used if you will generate the content for
```
Comment (paddatrapper) [inline docs/manual/howtos/build-schedule.md]: The ** should be before the space

File: docs/manual/howtos/build-schedule.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+a description of the steps involved in adding a show to the LibreTime
+calendar schedule for future playback including repeating shows
+
+Audience: Program Managers and Admins
+
+Click on the Calendar menu item on the left side of the screen
+
+Click on the blue New Show button
+
+Under **What – **type in a Name to replace Untitled Show
```
Comment (paddatrapper) [inline docs/manual/howtos/build-schedule.md]: Maybe the ** should be before ' -'?

File: docs/manual/howtos/build-schedule.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Optional – add URL, Genre and Description
+
+Under **When **click **In The Future - **
```
Comment (paddatrapper) [inline docs/manual/howtos/build-schedule.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/build-schedule.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+Select a date and time for the **Start Time** and **End Time **
```
Comment (paddatrapper) [inline docs/manual/howtos/build-schedule.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/build-schedule.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+Time is entered as 00:00 where the first 2 digits are the hour in
+military time and the second 2 digits are the minutes. So 15:00 is 3PM.
+
+Note: 24 hours is the maximum show time and Midnight is 00:00 of the
+next date.
+
+Click **Repeats **if this show will air more than once.
```
Comment (paddatrapper) [inline docs/manual/howtos/build-schedule.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/build-schedule.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+If this is a limited engagement show uncheck **No End?** and select the
+date at which this show should no longer be on the schedule.
+
+**Autoloading Playlist **is used if you will generate the content for
+this show using a playlist possibly containing smartblocks to
+dynamically generate it.
+
+Basically if this is checked and a playlist is selected LibreTime will
+schedule it an hour before the show is set to air as well as any **Intro
+& Outro Playlist **configured in the Admin settings.
```
Comment (paddatrapper) [inline docs/manual/howtos/build-schedule.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/intro-playlist.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+Click **+ New **
+
+Type **Intro Playlist **in the Name.
+
+Click **Smart Blocks**
+
+Click the box to the left of **Top of the Hour Station ID**
+
+Click **+ Add to current playlist **or drag the smart block to the right
```
Comment (paddatrapper) [inline docs/manual/howtos/intro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/intro-playlist.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+bottom of the playlist window.
+
+Click **Save**
+
+How to Set Default Intro Playlist
+
+Click **Settings**.
+
+Click General and then scroll to **Intro Autoloading Playlist **and
```
Comment (paddatrapper) [inline docs/manual/howtos/intro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/intro-playlist.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+click the drop down to select the **Intro Playlist **
```
Comment (paddatrapper) [inline docs/manual/howtos/intro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/intro-playlist.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+
+Click **Smart Blocks**
+
+Click the blue **+New** button
+
+On the right (or below if screen width is limited)
+
+Type in the Smart Block Name: **Top of the Hour Station ID**
+
+Under **Search Criteria **select **Genre **for the criteria
```
Comment (paddatrapper) [inline docs/manual/howtos/intro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/intro-playlist.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+
+Select **is **for modifier
```
Comment (paddatrapper) [inline docs/manual/howtos/intro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/livestream.md [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+
+**Audience**: DJs
+
+Download and install butt from <https://danielnoethen.de/> for your OS.
+
+Note: be sure you have butt version 0.1.17 or newer installed
+
+Open up butt – and click **settings**
+
+Under **Main **under **Server **click **ADD**
```
Comment (paddatrapper) [inline docs/manual/howtos/livestream.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/livestream.md [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+
+****
+
+Type LibreTime (or your station) under Name
+
+Click the radio button next to **IceCast** under Type
+
+Type your stations URL (webpage address) under **Address**:
+
+Type **8002 **under **Port**:
```
Comment (paddatrapper) [inline docs/manual/howtos/livestream.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/livestream.md [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+
+Type your DJ login password under **Password**
+
+Type **/show **under IceCast mountpoint:
```
Comment (paddatrapper) [inline docs/manual/howtos/livestream.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/livestream.md [hunk 15]
```diff
@@ -0,0 +1,89 @@
...
+
+You can record your show under butt by clicking the red circle record
+button on the left.
+
+It will save a mp3 based upon the date and time in your home/user
+directory by default.
+
+Everything should no be working and you can broadcast for your entire
+time slot. If you choose to stop streaming before it is over click the
+white square **Stop **button to disconnect.
```
Comment (paddatrapper) [inline docs/manual/howtos/livestream.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/outro-playlist.md [hunk 16]
```diff
@@ -0,0 +1,93 @@
...
+How to add Promo Fill smart block
+
+Click **Smart Blocks**
+
+Click the blue **+ New **button
+
+Type in a name for the Smart Block (for instance *Promo Fill*)
+
+Now we select **Genre **and the modifer of **is **and then type in
+**promo **in the 3^rd^ box.
```
Comment (paddatrapper) [inline docs/manual/howtos/outro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/outro-playlist.md [hunk 17]
```diff
@@ -0,0 +1,93 @@
...
+
+How to add Promo Fill smart block
+
+Click **Smart Blocks**
+
+Click the blue **+ New **button
+
+Type in a name for the Smart Block (for instance *Promo Fill*)
+
+Now we select **Genre **and the modifer of **is **and then type in
```
Comment (paddatrapper) [inline docs/manual/howtos/outro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/outro-playlist.md [hunk 18]
```diff
@@ -0,0 +1,93 @@
...
+**promo **in the 3^rd^ box.
+
+Under **Limit to**: we select **time remaining in show**
+
+To match on additional types of tracks based upon Genre
+
+click **+ New Modifier **under Genre and select **is** and type the
```
Comment (paddatrapper) [inline docs/manual/howtos/outro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/outro-playlist.md [hunk 19]
```diff
@@ -0,0 +1,93 @@
...
+“*name of genre*” to match on.
+
+Click **preview **to review and confirm tracks match your criteria
```
Comment (paddatrapper) [inline docs/manual/howtos/outro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/outro-playlist.md [hunk 20]
```diff
@@ -0,0 +1,93 @@
...
+below.
+
+Click **Save**
+
+How to add Ambient Fill that can be interrupted
+
+Click **Smart Blocks**
+
+Click the blue **+ New **button
```
Comment (paddatrapper) [inline docs/manual/howtos/outro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/outro-playlist.md [hunk 21]
```diff
@@ -0,0 +1,93 @@
...
+
+Type in a name for the Smart Block (for instance *Ambient Fill*)
+
+Now we select **Genre **and the modifer of **is **and then type in
```
Comment (paddatrapper) [inline docs/manual/howtos/outro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/outro-playlist.md [hunk 22]
```diff
@@ -0,0 +1,93 @@
...
+**ambient fill **in the 3^rd^ box.
```
Comment (paddatrapper) [inline docs/manual/howtos/outro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/outro-playlist.md [hunk 23]
```diff
@@ -0,0 +1,93 @@
...
+
+Under **Limit to**: we select **time remaining in show**
+
+Click **Advanced options **drop down
+
+Check box next to **Allow last track to exceed time limit**
+
+Click **preview **to review and confirm tracks match your criteria
```
Comment (paddatrapper) [inline docs/manual/howtos/outro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/outro-playlist.md [hunk 24]
```diff
@@ -0,0 +1,93 @@
...
+
+How to create the Outro Playlist to hold these
+
+Click **Playlists**
+
+Click **+ New**
+
+Type in the name “*Outro Playlist*”
+
+Click on **Smart Blocks **
```
Comment (paddatrapper) [inline docs/manual/howtos/outro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/outro-playlist.md [hunk 25]
```diff
@@ -0,0 +1,93 @@
...
+
+Click box next to **Promo Fill **smart block and click **+ Add to
```
Comment (paddatrapper) [inline docs/manual/howtos/outro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/outro-playlist.md [hunk 26]
```diff
@@ -0,0 +1,93 @@
...
+current playlist **or drag it to the bottom of the Outro playlist
```
Comment (paddatrapper) [inline docs/manual/howtos/outro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/outro-playlist.md [hunk 27]
```diff
@@ -0,0 +1,93 @@
...
+
+Click box next to **Ambient Fill **smart block and click **+ Add to
```
Comment (paddatrapper) [inline docs/manual/howtos/outro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/outro-playlist.md [hunk 28]
```diff
@@ -0,0 +1,93 @@
...
+current playlist **or drag it to the
```
Comment (paddatrapper) [inline docs/manual/howtos/outro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/outro-playlist.md [hunk 29]
```diff
@@ -0,0 +1,93 @@
...
+
+bottom of the Outro playlist
+
+Make sure the **Promo Fill **is above the **Ambient Fill** smart block
```
Comment (paddatrapper) [inline docs/manual/howtos/outro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/outro-playlist.md [hunk 30]
```diff
@@ -0,0 +1,93 @@
...
+in the playlist
+
+Click **Save **
```
Comment (paddatrapper) [inline docs/manual/howtos/outro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/outro-playlist.md [hunk 31]
```diff
@@ -0,0 +1,93 @@
...
+
+Setting the station-wide Outro playlist for all shows with auto loading
+playlists
+
+Click **Settings **
```
Comment (paddatrapper) [inline docs/manual/howtos/outro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/outro-playlist.md [hunk 32]
```diff
@@ -0,0 +1,93 @@
...
+
+Click **General **
```
Comment (paddatrapper) [inline docs/manual/howtos/outro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/outro-playlist.md [hunk 33]
```diff
@@ -0,0 +1,93 @@
...
+
+Scroll down to Outro Playlist and with the drop down select the newly
+created **Outro Playlist **
```
Comment (paddatrapper) [inline docs/manual/howtos/outro-playlist.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/podcast.md [hunk 34]
```diff
@@ -0,0 +1,57 @@
...
+
+Adding a podcast
+
+Steps:
+
+Login to LibreTime with a user with the Program Manager or Admin role
+
+Copy the URL to the podcast RSS feed
+
+Click on Podcasts and then click the blue **Add **button and then past
```
Comment (paddatrapper) [inline docs/manual/howtos/podcast.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/podcast.md [hunk 35]
```diff
@@ -0,0 +1,57 @@
...
+the feed URL and click Subscribe.
+
+Click the newly added Podcast and then click edit or double click it.
+
+Update the name of the podcast if you want to change it.
+
+On the right hand window should be open for your podcast click
+**Generate Smartblock and Playlist**
+
+Click the checkbox next the newest episode and then click the **import
```
Comment (paddatrapper) [inline docs/manual/howtos/podcast.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/upload-promo.md [hunk 36]
```diff
@@ -0,0 +1,44 @@
+***How to upload a promo***:
+
+Audience: DJs
+
+Instructions:
+
+Record your promo and export it as a mp3
+
+note: if you can set the metadata of genre to **promo **you can skip the
```
Comment (paddatrapper) [inline docs/manual/howtos/upload-promo.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/upload-promo.md [hunk 37]
```diff
@@ -0,0 +1,44 @@
...
+
+Click Upload and drag or click and browse to upload the track.
+
+Let the track upload. Once it is done and is shown as imported on the
+right click **Tracks**
+
+Find the track in the list. *(follow instructions below if you don’t see
+it)*
+
+If it is not at the top click the **Uploaded **column until it sorts by
```
Comment (paddatrapper) [inline docs/manual/howtos/upload-promo.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/upload-promo.md [hunk 38]
```diff
@@ -0,0 +1,44 @@
...
+the newest tracks first.
+
+If Uploaded doesn’t show up click **Columns **to the top right scroll
```
Comment (paddatrapper) [inline docs/manual/howtos/upload-promo.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/upload-promo.md [hunk 39]
```diff
@@ -0,0 +1,44 @@
...
+down and click the box next to **Uploaded**
+
+****
+
+Now click **Uploaded **twice and it should display your track at the
```
Comment (paddatrapper) [inline docs/manual/howtos/upload-promo.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/upload.md [hunk 40]
```diff
@@ -0,0 +1,58 @@
+*How to upload and schedule a track to play on a specific show*
+
+Audience: DJ with permission to schedule a specific show
+
+**Login **to LibreTime with your DJ username and password at
```
Comment (paddatrapper) [inline docs/manual/howtos/upload.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/upload.md [hunk 41]
```diff
@@ -0,0 +1,58 @@
...
+[http://](http://HOSTNAME/user)[**HOSTNAME**](http://HOSTNAME/user)[/user](http://HOSTNAME/user)
```
Comment (paddatrapper) [inline docs/manual/howtos/upload.md]: This looks odd... Why is each element linked separately?

File: docs/manual/howtos/upload.md [hunk 42]
```diff
@@ -0,0 +1,58 @@
...
+-
+
+note: replace HOSTNAME with the web site address given to you by your
+station admin
+
+Click the blue **Upload** button in the top left of the screen.
+
+Click the background where it says **Drop files here or click to browse
+your computer **or dragon and drop your music files from your desktop.
```
Comment (paddatrapper) [inline docs/manual/howtos/upload.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/upload.md [hunk 43]
```diff
@@ -0,0 +1,58 @@
...
+
+If you clicked then browse your file system to find the music files you
+want to play and click them and then click **Open**
+
+The tracks should start to upload and when they are completed show up on
+the right under **Recent Uploads ***(note: you will need to keep this
```
Comment (paddatrapper) [inline docs/manual/howtos/upload.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/upload.md [hunk 44]
```diff
@@ -0,0 +1,58 @@
...
+
+**
+
+Once you have uploaded all of the tracks you plan on airing you click on
+**Calendar**
+
+Find your upcoming show in the Calendar and click on it and click
+**Schedule Tracks** from the pop-up menu.
+
+Now click the tracks you want to add and click **+ Add to show **or
```
Comment (paddatrapper) [inline docs/manual/howtos/upload.md]: Spacing is causing issue with Markdown formatting

File: docs/manual/howtos/upload.md [hunk 45]
```diff
@@ -0,0 +1,58 @@
...
+You can also add station **Smart Blocks, Playlists and Webstreams by
+clicking those sections at the top of the screen.**
+
+****
+
+When you are done you can re-order your tracks by clicking and dragging
+them.
+
+When your tracks are in order and there is no empty time remaning click
+**Ok **to finish.
```
Comment (paddatrapper) [inline docs/manual/howtos/upload.md]: Spacing is causing issue with Markdown formatting

File: mkdocs.yml [hunk 46]
```diff
@@ -22,6 +22,8 @@ pages:
   - 'Using LibreTime':
     - 'On Air in 60 seconds!': 'manual/on-air-in-60-seconds/index.md'
     - 'Getting Started': manual/getting-started/index.md
+    - 'Tutorials' : manual/tutorials/index.md
```
Comment (paddatrapper) [inline mkdocs.yml]: Just for consistency, can the `:` be directly after the key and the value be bounded by single quotation marks?

File: mkdocs.yml [hunk 47]
```diff
@@ -22,6 +22,8 @@ pages:
...
+    - 'How-Tos' : manual/howtos/index.md
```
Comment (paddatrapper) [inline mkdocs.yml]: Consistency here too

**Comment by Robbt** [comment]
Ok, most of that formatting stuff happened in the pandoc conversion from the openoffice where I originally wrote them. But I attempted to correct all of the issues in my latest commit. So this should be good to merge. I'd like to do more formatting at some point and possibly add screenshots as I think this would make it easier to follow. But they also closely mirror the various tutorials I built.

**Pull request merged**